react-components/src/queries/query_cart_details.graphql

```diff
@@ -1,5 +1,6 @@
 query cartDetails($cartId: String!) {
     cart(cart_id: $cartId) {
+        is_virtual
```

Comment:
Is this `true` only when all the cart products are virtual?

Reply:
Yes. As soon as one physical item is in the cart, this property is `false`.